src/lib/RadworksGovernorTimelockCompound.sol

```diff
@@ -0,0 +1,247 @@
+// SPDX-License-Identifier: MIT
+// OpenZeppelin Contracts (last updated v4.6.0) (governance/extensions/GovernorTimelockCompound.sol)
+//
+// This RadworksGovernorTimelockCompound has been modified from the OZ GovernorTimelockCompound to
+// support a Radworks Timelock which does not conform to Governor Alpha Timelock. If Radworks
+// decides to change the Timelock in the future then it must conform to the Radworks Timelock
+// interface.
```

Comment:
Is the existing timelock acceptable enough?

Reply:
Agree: c7f59d45d8da2cc21cee3ad4d502f59eb9dfc04f

Reply:
ah, ok! so radworks timelock is simply the default? this simplifies things :+1:

Reply:
Yes, it is different, but it doesn't conflict in the same way as the poolTogether timelock did.